src/utils/files.ts

```diff
@@ -34,3 +34,10 @@ export const writeFile = (filePath: string, data: string | NodeJS.ArrayBufferVie
   // Then write file
   fs.writeFileSync(filePath, data, "utf-8");
 };
+
+export const createSymlink = (targetPath: string, linkPath: string, type: "file" | "dir" | "junction" = "file") => {
```

Comment:
Is this called anywhere?

Reply:
Same. These are utils that modules use. Like common functionality that modules usually require.